apps/admin-x-activitypub/src/components/feed/FeedItem.tsx

```
@@ -276,11 +284,15 @@ const FeedItem: React.FC<FeedItemProps> = ({actor, object, layout, type, comment
                                             </div>
                                         </div> :
                                             <div className='relative'>
-                                                <div
-                                                    dangerouslySetInnerHTML={({__html: object.content ?? ''})}
-                                                    ref={contentRef}
-                                                    className='ap-note-content line-clamp-[10] text-pretty leading-[1.4285714286] tracking-[-0.006em] text-grey-900'
-                                                ></div>
+                                                <div className='ap-note-content line-clamp-[10] text-pretty leading-[1.4285714286] tracking-[-0.006em] text-grey-900'>
+                                                    {!isLoading ?
+                                                        <div dangerouslySetInnerHTML={{
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)